rasa/core/nlg/template.py

```diff
@@ -110,13 +110,17 @@ def _fill_template(
             "image",
             "custom",
             "button",
+            "buttons",
```

Comment:
```suggestion
            "buttons",
```

Reply:
that should be the only change you need (+ fixing the test cases)

Reply:
Hello, Have done these

Reply:
anything else?